certification/internal/shell/validate_operator_package_name_uniq_test.go

```diff
@@ -0,0 +1,30 @@
+package shell
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
+)
+
+var _ = Describe("BundlePackageNameUniqCheck", func() {
+	var (
+		bundlePackageNameUniqCheck ValidateOperatorPkNameUniqCheck
+	)
+
+	Describe("Operator Bundle Package Name Uniqueness Validate", func() {
```

Comment:
out of curiosity: I don't see the external call being mocked in BeforeEach. I am wondering if you don't need to mock the API call to Pyxis?